deploy/complete/helm-chart/mushop/charts/fulfillment/templates/fulfillment-svc.yaml

```diff
@@ -2,16 +2,15 @@ apiVersion: v1
 kind: Service
 metadata:
 # TODO: update to shipping.fullname when orders can receive variable
-  name: {{ include "shipping.name" . }}
-  labels: {{ include "shipping.labels" . | nindent 4 }}
+  name: {{ include "fulfillment.name" . }}
+  labels: {{ include "fulfillment.labels" . | nindent 4 }}
 spec:
-  type: NodePort
   selector:
-    app.kubernetes.io/name: {{ include "shipping.name" . }}
+    app.kubernetes.io/name: {{ include "fulfillment.name" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
-    run: {{ include "shipping.name" . }}
+    run: {{ include "fulfillment.name" . }}
   ports:
     - port: {{ .Values.service.port }}
       name: http
       protocol: TCP
-      targetPort: 80
+      targetPort: 8099
```

Comment:
8099? It's possible to keep 80 as the other services?

Reply:
No all the services uses 80 to listen on. Fulfillment service exposes 80. the app simply listens on 8099.
I can change, but is it really required ?

Reply:
Not required. Just asking if would have a specific reason. The targetPort from other service point to 80 as standard, except api, assets, user ... never mind, we do not have a standard (all odd port are from NodeJS base services).